### DISCUSSION TITLE
Where are the Server settings? ie..SMTP

### DISCUSSION THREAD

**Discussion by tharris9d** [vanjarosoftware/Vanjaro.Platform#1841] [General]
I am sure I  am just missing something. Where do I find the server settings for setting SMTP etc? Also is there a way to add a standard dnn page with persona bar in Vanjaro 1.1?

**Comment by Tychodewaard** [comment]
![image](https://user-images.githubusercontent.com/9091997/136260431-586afa26-63ec-4c0f-b42f-51000674222b.png)
Type SMTP in the search box.

If you add a page using vanjaro, you create a Vanjaro page. If you want to create a DNN page, it can still be done if you run Vanjaro on DNN and go to /admin. You'll find the persona bar and all the things you are used to.

**Reply by tharris9d** [reply]
Thanks. I found the SMTP following your tip. 

I installed Vanjaro platform not the dnn Vanjaro extention. Can a dnn page be added?

**Reply by meetmandeep** [reply]
No. If you need to run both DNN and Vanjaro together then you need to install Vanjaro for DNN on an existing DNN Install.

**Reply by tharris9d** [reply]
All the email is queued. I checked the database and can see 20 queued emails. How do we send them?

**Reply by mksmanish79** [reply]
Check if the scheduler is running

**Reply by tharris9d** [reply]
Scheduler is running. Messages sent 0
![EmailQueue_dbo VJ_Common_MailQueue](https://user-images.githubusercontent.com/217445/136587653-e340636b-3d53-481c-88ae-80f037acd32f.png)
![Messaging Dispatch](https://user-images.githubusercontent.com/217445/136587713-97563a08-91ec-4b24-98bb-912b7cc39345.png)

**Reply by tharris9d** [reply]
There are no errors showing in the logs.

**Reply by meetmandeep** [reply]
It's likely the issue reported in #1565 

It has been fixed in v1.2.0 which just released.

**Comment by tharris9d** [comment]
I upgraded to 1.2, Clear server and client caches, cleared cookies and restarted application.
It's not working. I am getting this exception in the log. I am using the same working SMTP settings I use in DNN.

General Exception

AbsoluteURL DefaultDataProvider DotNetNuke.Data.SqlDataProvider DotNe

AbsoluteURL:

DefaultDataProvider:DotNetNuke.Data.SqlDataProvider, DotNetNuke

ExceptionGUID:4c1afc63-24b0-47ab-96eb-c52e44a27426

AssemblyVersion:

PortalId:-1

UserId:-1

TabId:-1

RawUrl:

Referrer:

UserAgent:

ExceptionHash:IARkitwahaHXvB5ygJS4WyrcDYs=

Message:Object reference not set to an instance of an object.

StackTrace:

at Vanjaro.Core.Providers.Mail.VanjaroMailProvider.SendMail(MailInfo mailInfo, SmtpInfo smtpInfo)
InnerMessage:

InnerStackTrace:

Source:Vanjaro.Core.Providers.Mail.VanjaroMailProvider

FileName:

FileLineNumber:0

FileColumnNumber:0

Method:

**Reply by mksmanish79** [reply]
Can you confirm if you have specified port no in **Server and port** setting?

**Reply by mksmanish79** [reply]
How frequently the **Object reference not set to an instance of an object is logged**? Any scenario when the issue is logged?
